src/parity_testing/auth/testSuites/signOut/Test_that_globalSignOut_returns_partial_success_with_revoke_token_error.json

```diff
@@ -34,13 +34,14 @@
         {
             "type": "amplify",
             "apiName": "signOut",
-            "responseType": "complete",
```

Comment:
As discussed during the test harness design, `responseType` can be either `failure` or `success`.